ddtrace/contrib/boto/patch.py

```diff
@@ -85,8 +87,10 @@ def patched_query_request(original_func, instance, args, kwargs):
     endpoint_name = getattr(instance, "host").split(".")[0]
 
     with pin.tracer.trace(
-        "{}.command".format(endpoint_name),
-        service="{}.{}".format(pin.service, endpoint_name),
+        schematize_cloud_api_operation(
+            "{}.command".format(endpoint_name), cloud_provider="aws", cloud_service=endpoint_name
+        ),
+        service=schematize_service_name("{}.{}".format(pin.service, endpoint_name)),
```

Comment:
We are schematizing the default service names for integrations here.  In othe words, manual spans and manually specified overrides still take precedence.

The priority order after this change is (generally, not in all cases):

1. Manual service name
2. DD_INTEGRATION_SERVICE env var
3. DD_INTEGRATION_SERVICE_NAME env var
4. DD_SERVICE
5. "unnamed-python-service"

If any level evaluates to "False" in an assert statement, then we go down one.

Reply:
> It might be easier if we had one helper in tracer._start_span() that sets the expected service and operation names. 

This is difficult with the current architecture, as every config determines the service name separately, and we want to only schematize a span if its using the default values.  By the time the span is initialized, we've lost all the context about whether or not the passed in service name is a default or if its been provided/overidden by a user.
 
> We will be changes the core api but it will be easier to maintain. I can see contributors creating integrations and forgetting to use these helpers.

I wholeheartedly agree with this, and this is actually a problem across all tracers.  I'd like to decouple this effort with a refactoring effort.  With that said, I am writing this down as a problem